__tests__/log.test.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @jest-environment jsdom
```

Comment:
Can we have a test with a node environment?

Reply:
added a copy of this but with node env, wondering if there is a better way to run the same test in different envs

Reply:
Copy is fine for now